res/openvpn-gui-res-en.rc

```diff
@@ -254,10 +254,10 @@ Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n\
 Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n\
 Copyright (C) 2012-2022 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
     LTEXT "https://github.com/OpenVPN/openvpn-gui/", ID_LTEXT_ABOUT2, 36, 55, 206, 42
-    LTEXT "OpenVPN – An application to securely tunnel IP networks \
-over a single TCP/UDP port, with support for SSL/TLS-based \
-session authentication and key exchange, packet \
-encryption, packet authentication, and packet compression.\n\
+    LTEXT "OpenVPN GUI - برنامه ای برای تونل ایمن شبکه های  IP \
+از طریق یک پورت  TCP/UDP، با پشتیبانی از جلسه  \
+احراز هویت مبتنی بر SSL/TLS و تبادل کلید، رمزگذاری بسته ها، \
+احراز هویت بسته ها، و فشرده سازی بسته ها. \n\
```

Comment:
This is the English version, do not translate it!

Reply:
ok

Reply:
> > @jvdi You had provided Farsi translations in the past. Could you take a look at this PR please?
> 
> That is a good translation
> 
> If app can support RTL direction for show this text, because Farsi text and English text in one line have some problem → if direction of the text not be RTL.
> 
> ![1](https://user-images.githubusercontent.com/40993115/205459830-d8fd22e5-4757-4047-9db2-8cd36bc30aa0.gif) [ ![1](https://user-images.githubusercontent.com/40993115/205459830-d8fd22e5-4757-4047-9db2-8cd36bc30aa0.gif) ](https://user-images.githubusercontent.com/40993115/205459830-d8fd22e5-4757-4047-9db2-8cd36bc30aa0.gif) [ ](https://user-images.githubusercontent.com/40993115/205459830-d8fd22e5-4757-4047-9db2-8cd36bc30aa0.gif)

@selvanair did this , It's RTL now.

Reply:
![image](https://user-images.githubusercontent.com/118456291/205460103-2e44e1d5-13ac-4216-aab5-d57ee0bafbaa.png)